Makefile

```diff
@@ -345,9 +344,24 @@ endif
 CLUSTER_OP_ANSIBLE_REPO   ?= https://github.com/openshift/openshift-ansible.git
 CLUSTER_OP_ANSIBLE_BRANCH ?= master
 
+define build-cluster-operator-ansible-image #(repo, branch, imagename, tag)
+	echo "repo $1"
+	echo "branch $2"
+	echo "image $3:$4"
+	docker build -t "$3:$4" --build-arg=CO_ANSIBLE_URL=$1 --build-arg=CO_ANSIBLE_BRANCH=$2 build/cluster-operator-ansible
```

Comment:
These pre-existing vars could possibly use a rename for clarity, this isn't the location of CO ansible, it's the openshift-ansible ansible, maybe OA_ANSIBLE_URL and BRANCH?